packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -150,26 +202,44 @@ class HorizontalOverflow extends Foundation<
     }
 
     /**
-     * Callback on scroll change
+     * Callback for on scroll change
      */
     private onScrollChange = (): void => {
-        if (!this.props.onScrollChange) {
-            return;
+        if (typeof this.props.onScrollChange === "function") {
+            this.props.onScrollChange(this.getScrollChangeData());
+        }
+
+        // If the onOverflowChange callback exist, we want to update overflow
+        // based on scroll change
+        if (typeof this.props.onOverflowChange === "function") {
+            const positionData: ScrollChange = this.getScrollChangeData();
+
+            if (
+                this.overflowStart === positionData.start ||
+                this.overflowEnd === positionData.end
+            ) {
+                this.handleOverflowChange();
+            }
         }
+    };
 
+    /**
+     * Get the scroll change data
+     */
+    private getScrollChangeData = (): ScrollChange => {
```

Comment:
Can we change this name since it is being used by both the scroll and overflow callbacks to reflect the data it's actually getting back, I think `getItemsPositionWithinContainer` unless there is a better name we can come up with.

Reply:
Yeah I had the same thought on changing this name. @nicholasrice for input as well.

Reply:
Agreed on `getPositionData`